controllers/configurationpolicy_utils.go

```diff
@@ -540,18 +541,19 @@ func configPlcHasFinalizer(plc policyv1.ConfigurationPolicy, finalizer string) b
 	return false
 }
 
-func addConfigPlcFinalizer(plc policyv1.ConfigurationPolicy, finalizer string) []string {
-	if configPlcHasFinalizer(plc, finalizer) {
-		return plc.GetFinalizers()
+func addObjFinalizer(obj metav1.Object, finalizer string) []string {
+	if objHasFinalizer(obj, finalizer) {
+		return obj.GetFinalizers()
 	}
 
-	return append(plc.GetFinalizers(), finalizer)
+	return append(obj.GetFinalizers(), finalizer)
 }
 
-func removeConfigPlcFinalizer(plc policyv1.ConfigurationPolicy, finalizer string) []string {
+// nolint: unparam
```

Comment:
Is this nolint actually needed?

Reply:
The linter complained about it... We only have one finalizer, so it makes sense. The real mystery is why the linter doesn't complain about the other (similar) functions with this issue.

Reply:
Oh, you think it's detecting that a constant is always passed as its parameter?